prisma/migrations/20231130123148_create_media_table/migration.sql

```diff
@@ -0,0 +1,12 @@
+-- CreateTable
+CREATE TABLE "Media" (
+    "id" UUID NOT NULL DEFAULT gen_random_uuid(),
+    "url" TEXT NOT NULL,
+    "filename" TEXT NOT NULL,
+    "contentType" TEXT NOT NULL,
```

Comment:
is this extension or different like application/png?

Reply:
its mime like image/png, application/pdf, etc